package.json

```diff
@@ -36,8 +36,12 @@
     "test": "ember test",
     "test:all": "ember try:each"
   },
+  "peerDependencies": {
+    "ember-source": "*"
+  },
```

Comment:
FYI for other's coming along later, this is required in order to work properly with `dependencySatisifies` from `@embroider/macros`.